tests/nested/manual/core20-save/task.yaml

```diff
@@ -0,0 +1,91 @@
+summary: Check that ubuntu-save is set up in a UC20 device
+
+description: |
+    This test checks that ubuntu-save is preset and set up correctly in a UC20
+    device or that a device can boot when ubuntu-save is not required
+
+# this is a UC20 specific test
+systems: [ubuntu-20.04-64]
+
+environment:
+  # ubuntu-save, TPM, secure boot
+  NESTED_ENABLE_TPM/encrypted: true
+  NESTED_ENABLE_SECURE_BOOT/encrypted: true
+  NESTED_ADD_UBUNTU_SAVE/encrypted: true
+
+  # add ubuntu-save, but but no TPM or secure boot support
+  NESTED_ENABLE_TPM/plainwithsave: false
+  NESTED_ENABLE_SECURE_BOOT/plainwithsave: false
+  NESTED_ADD_UBUNTU_SAVE/plainwithsave: true
+
+  # no ubuntu-save, no TPM, no secure boot
+  NESTED_ENABLE_TPM/plain: false
+  NESTED_ENABLE_SECURE_BOOT/plain: false
+  NESTED_ADD_UBUNTU_SAVE/plain: false
+
+prepare: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
```

Comment:
No need to source this

Reply:
Ahh right, we got the *-state tools now.